dashboard/test/ui/test_runner/test_runner_config_variables.rb

```diff
@@ -0,0 +1,4 @@
+module TestRunnerConfigVariables
```

Comment:
I'm not sure "config variables" is accurate for these values; what about "constants"?